configure.ac

```diff
@@ -2827,6 +2829,560 @@ if test x$host_win32 = xno; then
 		AC_CHECK_HEADERS(sys/random.h)
 		AC_CHECK_FUNCS(getrandom getentropy)
 	fi
+
+	dnl *********************************
+	dnl *** Checks for System.Native  ***
+	dnl *********************************
+
+	AC_MSG_CHECKING(for linux/in.h)
+	AC_TRY_COMPILE([
+		#include <sys/socket.h>
+		#include <linux/in.h>
+	], [
+	],[
+		AC_MSG_RESULT(yes)
+		AC_DEFINE(HAVE_LINUX_IN_H, 1, [linux/in.h])
+	], [
+		AC_MSG_RESULT(no)
+	])
+
+	AC_MSG_CHECKING(for struct in_pktinfo)
+	AC_TRY_COMPILE([
+		#ifdef HAVE_LINUX_IN_H
+		#include <sys/socket.h>
+		#include <linux/in.h>
+		#else
+		#include <netinet/in.h>
+		#endif
+	], [
+		struct in_pktinfo pktinfo;
+	],[
+		AC_MSG_RESULT(yes)
+		AC_DEFINE(HAVE_IN_PKTINFO, 1, [struct in_pktinfo])
+	], [
+		AC_MSG_RESULT(no)
+	])
+
+	AC_MSG_CHECKING(for struct ip_mreqn)
+	AC_TRY_COMPILE([
+		#if HAVE_LINUX_IN_H
+		#include <sys/socket.h>
+		#include <linux/in.h>
+		#else
+		#include <netinet/in.h>
+		#endif
+	], [
+		struct ip_mreqn mreqn;
+	],[
+		AC_MSG_RESULT(yes)
+		AC_DEFINE(HAVE_IP_MREQN, 1, struct ip_mreqn)
+	], [
+		AC_MSG_RESULT(no)
+	])
+
+	if test x$host_darwin = xno; then
+		AC_MSG_CHECKING(for struct flock64)
+		AC_TRY_COMPILE([
+			#include <fcntl.h>
+		], [
+			struct flock64;
+		],[
+			AC_MSG_RESULT(yes)
+			AC_DEFINE(HAVE_FLOCK64, 1, struct flock64)
+		], [
+			AC_MSG_RESULT(no)
+		])
+	fi
+
+	AC_CHECK_FUNC(lseek64,           [AC_DEFINE(HAVE_LSEEK64, 1, [lseek64])])
+	AC_CHECK_FUNC(mmap64,            [AC_DEFINE(HAVE_MMAP64, 1, [mmap64])])
```

Comment:
Using AC_CHECK_FUNCS(....) would be simpler.

Reply:
Personally I like a line per "thing" for easier to see diffs (whatever is a "thing"). Instead of edits to long lines where you have to find the intra-line diff. For example there are coding conventions with a parameter per line. But conserving vertical space is also a goal.

I suspect this got copied from elsewhere.